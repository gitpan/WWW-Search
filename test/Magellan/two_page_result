http://www.mckinley.com/search.gw?look=magellan&start=0&search=%2BMartin+%2BThurn&
http://www.mckinley.com/search.gw?look=magellan&start=10&search=%2BMartin+%2BThurn&
