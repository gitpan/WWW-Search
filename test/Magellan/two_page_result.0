<HTML>
<HEAD><TITLE>Search Results</TITLE></HEAD>
<BODY BGCOLOR="#ffffff">

<CENTER>
<TABLE BORDER=0 CELLPADDING=0 CELLSPACING=2>
 <TR>
<TD ALIGN=LEFT VALIGN=top ROWSPAN=4>
<A HREF="http://www.mckinley.com/">
<IMG SRC="/img/magellan/mglobe.gif" BORDER=0 WIDTH=49 HEIGHT=41 ALIGN=left></A>
</TD>
<TD ALIGN=middle VALIGN=top ROWSPAN=4>
<A HREF="http://www.mckinley.com/"><IMG SRC="/img/magellan/magellan.gif" BORDER=0 WIDTH=224 HEIGHT=41 ALIGN=middle ALT="Magellan Internet Guide"></A><BR>
<IMG SRC="/img/magellan/white.gif" HEIGHT=4 WIDTH=1><BR>
<FONT FACE="arial,helvetica" SIZE="-2">
<A HREF="http://www.mckinley.com/">HOME</A> | <A HREF="/magellan/Reviews/index.magellan.html">REVIEWS</A></FONT>
</TD><TD WIDTH=4 ROWSPAN=4></TD>
<TD HEIGHT="2" COLSPAN=3 NOWRAP></TD></TR>

<TR>


<FORM ACTION="/search.gw" METHOD=get>
<TD ALIGN=left VALIGN=top COLSPAN=3 NOWRAP>
<INPUT NAME=search VALUE="+ISI +divisions" SIZE=30>
<INPUT TYPE=IMAGE SRC="/img/magellan/searchbut.gif" BORDER=0 ALIGN="top" VALUE="Search Magellan" WIDTH=67 HEIGHT=30>
</TD></TR>
<TR>
 <TD ALIGN=left NOWRAP>
 <INPUT TYPE=radio NAME=c VALUE=web.guide
    > <FONT SIZE="-1">Reviewed Sites Only</FONT></TD>
 <TD WIDTH=3 ROWSPAN=2></TD>
 <TD ALIGN=left>
  <INPUT TYPE=radio NAME=c VALUE=web.green
        ><FONT SIZE="-1" COLOR=#008C05>Green Light</FONT><FONT SIZE="-1"> sites only</FONT>
 </TD>
</TR>
<TR>
 <TD ALIGN=left><INPUT TYPE=radio NAME=c VALUE=web
   CHECKED  > <FONT SIZE="-1">The Entire Web</FONT>
<INPUT TYPE=hidden NAME=look VALUE=magellan>
</TD>

<!-- End of search and collection section -->


<TD ALIGN=left><FONT SIZE="-1"><A HREF="/magellan/Info/advancedtips.html">Search Tips and Help</A></FONT></TD></TR></TABLE>

<input type=hidden name=sorig value=rpage>
</FORM>
</CENTER>
<P>


 <CENTER><!-- Ad Start -->
<!-- Ad param svcpage=/search.gw keyword=isi+divisions svccobrand=magellan svcname=xsr ip=128.9.160.184 user_agent=WWW::Search/1.023 -->
<a href="/relocate/ad=61447_excite1023_search2celt-468x60.gif|pk=1564947;http://www.excite.com/info/company.html" onMouseOver="window.status='Click Here !';return true" language="Javascript"><img src="http://image.eimg.com/ads/61447_excite1023_search2celt-468x60.gif" ALT="Click Here !" HEIGHT=60 WIDTH=468 BORDER=1><CENTER><FONT SIZE=-1 FACE=geneva,arial COLOR=0000FF> <B> Click Here ! </B></FONT></CENTER></a>
<!-- Ad Stop -->
</CENTER>

<BLOCKQUOTE>

<B>14</B> results returned, ranked by relevance.

<!-- IF statement for Green Light text -->



<FORM ACTION="/search.gw" METHOD=get>
    <INPUT TYPE=hidden NAME=search VALUE="+ISI +divisions">
    <INPUT TYPE=hidden NAME=look VALUE=magellan>
             <INPUT TYPE=submit NAME=next VALUE="Next Results">
          <INPUT TYPE=hidden NAME=start VALUE=0>
    <INPUT TYPE=hidden NAME=collection VALUE=web>
            
<!--search results-->

   <HR>
  <B><A HREF="http://www.isi.edu/">USC Information Sciences
Institute</A></B>&nbsp;&nbsp;&nbsp;
  <B>75%</B>&nbsp;&nbsp;&nbsp;
  <A HREF="/search.gw?lk=magellan&c=web&qbe=304592&ctag=3664781D&showSummary=true"><font size="-1">[find similar]</A></FONT><BR>
      This web site requires a browser that supports HTML tables and jpeg files for proper viewing!  Comments and Questions regarding this site may be sent to action@isi.edu.   
    <BR>
  <I> <A HREF="http://www.isi.edu/">http://www.isi.edu/</A></I>
  <BR>
   <HR>
  <B><A HREF="http://www.isitrans.com/">ISI, Your Language Solution</A></B>&nbsp;&nbsp;&nbsp;
  <B>67%</B>&nbsp;&nbsp;&nbsp;
  <A HREF="/search.gw?lk=magellan&c=web&qbe=296105&ctag=3664781D&showSummary=true"><font size="-1">[find similar]</A></FONT><BR>
      ISI provides language support for translation and localization in any platform for any product, documentation or software; we can even help you with your multilingual WEB page including HTML support.  
    <BR>
  <I> <A HREF="http://www.isitrans.com/">http://www.isitrans.com/</A></I>
  <BR>
   <HR>
  <B><A HREF="http://www.nesconsult.com/">Welcome to NES Inc., We Make A Difference
</A></B>&nbsp;&nbsp;&nbsp;
  <B>66%</B>&nbsp;&nbsp;&nbsp;
  <A HREF="/search.gw?lk=magellan&c=web&qbe=629755&ctag=3664781D&showSummary=true"><font size="-1">[find similar]</A></FONT><BR>
      Providing Clients with Engineering Services, Custom Engineered Products and Construction Remediation Services  For more than twenty years, NES, Inc. has provided services and products to save clients money by:   
    <BR>
  <I> <A HREF="http://www.nesconsult.com/">http://www.nesconsult.com/</A></I>
  <BR>
   <HR>
  <B><A HREF="http://www.westernreserve.com/">Western Reserve Life</A></B>&nbsp;&nbsp;&nbsp;
  <B>65%</B>&nbsp;&nbsp;&nbsp;
  <A HREF="/search.gw?lk=magellan&c=web&qbe=1932804&ctag=3664781D&showSummary=true"><font size="-1">[find similar]</A></FONT><BR>
      Revised: Wednesday, October 18, 1998  * MEMBERSHIP PROMOTES ETHICAL MARKET CONDUCT FOR INDIVIDUAL LIFE INSURANCE AND ANNUITIES Western Reserve Life Assurance Co. of Ohio (&quot;WRL&quot;) is licensed to do business as a life insurance company in the District of Columbia and all states of the U.S. except New York.   
    <BR>
  <I> <A HREF="http://www.westernreserve.com/">http://www.westernreserve.com/</A></I>
  <BR>
   <HR>
  <B><A HREF="http://www.detnews.com/1996/menu/poli0904.htm">Election  96</A></B>&nbsp;&nbsp;&nbsp;
  <B>61%</B>&nbsp;&nbsp;&nbsp;
  <A HREF="/search.gw?lk=magellan&c=web&qbe=516089&ctag=3664781D&showSummary=true"><font size="-1">[find similar]</A></FONT><BR>
      Jack Kemp's message isn't one that Robbin Coleman says he is used to hearing from Republicans.  The Republican vice-presidential candidate showed up at ISI Robotics Inc. in Fraser Tuesday where Coleman works as a tool-crib attendant to talk about healing the divisions between the races, between labor and management and giving the poor an opportunity.   
    <BR>
  <I> <A HREF="http://www.detnews.com/1996/menu/poli0904.htm">http://www.detnews.com/1996/menu/poli0904.htm</A></I>
  <BR>
   <HR>
  <B><A HREF="http://hsb.baylor.edu/credits.asp">Site Design Credits</A></B>&nbsp;&nbsp;&nbsp;
  <B>56%</B>&nbsp;&nbsp;&nbsp;
  <A HREF="/search.gw?lk=magellan&c=web&qbe=1965046&ctag=3664781D&showSummary=true"><font size="-1">[find similar]</A></FONT><BR>
      HSB News Search Sitemap Help Master Calendar Faculty Profiler Request More Information Site Design Credits  Site Design Credits  The design of Baylor University's Hankamer School of Business web site was a joint venture utilizing the talents of local web design firm SharpSite Interactive, an ISI Management Company and Hankamer graduate student, Rob Stewart.   
    <BR>
  <I> <A HREF="http://hsb.baylor.edu/credits.asp">http://hsb.baylor.edu/credits.asp</A></I>
  <BR>
   <HR>
  <B><A HREF="http://members.tripod.com/~bud_lite/buds_resume.html">Robert Bud James' Resume (Information Technology Team Leader)</A></B>&nbsp;&nbsp;&nbsp;
  <B>47%</B>&nbsp;&nbsp;&nbsp;
  <A HREF="/search.gw?lk=magellan&c=web&qbe=1844379&ctag=3664781D&showSummary=true"><font size="-1">[find similar]</A></FONT><BR>
      Robert Bud James' Resume  
    <BR>
  <I> <A HREF="http://members.tripod.com/~bud_lite/buds_resume.html">http://members.tripod.com/~bud_lite/buds_resume.html</A></I>
  <BR>
   <HR>
  <B><A HREF="http://grognard.com/patch.html">Web Grognards: patches etc for computer wargames</A></B>&nbsp;&nbsp;&nbsp;
  <B>47%</B>&nbsp;&nbsp;&nbsp;
  <A HREF="/search.gw?lk=magellan&c=web&qbe=298028&ctag=3664781D&showSummary=true"><font size="-1">[find similar]</A></FONT><BR>
      Note - if you have problems unpacking any of the following files try using Fetch (version 2.1.1 or later) to get them using anonymous FTP from grognard.com/pub/comp/mac/.   
    <BR>
  <I> <A HREF="http://grognard.com/patch.html">http://grognard.com/patch.html</A></I>
  <BR>
   <HR>
  <B><A HREF="http://www.gospelcom.net/shofar/untilv4i6.html">Until Prophecy Newsletter</A></B>&nbsp;&nbsp;&nbsp;
  <B>47%</B>&nbsp;&nbsp;&nbsp;
  <A HREF="/search.gw?lk=magellan&c=web&qbe=1595183&ctag=3664781D&showSummary=true"><font size="-1">[find similar]</A></FONT><BR>
      PROPHECY NEWSLETTER Volume 4 Issue 6 Table of Contents:  Jews Mark High Holy Days / Prophetic Prospective on the Holidays Israel's Intrigue an Insult / Intrigue's PropheticProspective  
    <BR>
  <I> <A HREF="http://www.gospelcom.net/shofar/untilv4i6.html">http://www.gospelcom.net/shofar/untilv4i6.html</A></I>
  <BR>
   <HR>
  <B><A HREF="http://chaos.fullerton.edu/mhslinks.html">Particle-Surface Resources on the Internet</A></B>&nbsp;&nbsp;&nbsp;
  <B>45%</B>&nbsp;&nbsp;&nbsp;
  <A HREF="/search.gw?lk=magellan&c=web&qbe=1046481&ctag=3664781D&showSummary=true"><font size="-1">[find similar]</A></FONT><BR>
      This page is maintained by Dr.  Mark H.  Shapiro at the Department of Physics, California State University, Fullerton.  The information is believed to be accurate; however, use of the information is at your own risk.   
    <BR>
  <I> <A HREF="http://chaos.fullerton.edu/mhslinks.html">http://chaos.fullerton.edu/mhslinks.html</A></I>
  <BR>
 <HR>
</FORM>
<FORM ACTION="/search.gw" METHOD=get>
             <INPUT TYPE=submit NAME=next VALUE="Next Results">
       <INPUT TYPE=hidden NAME=search VALUE="+ISI +divisions">
    <INPUT TYPE=hidden NAME=look VALUE=magellan>
       <INPUT TYPE=hidden NAME=start VALUE=0>
    <INPUT TYPE=hidden NAME=collection VALUE=web>
          </FORM>

Try your search on:&nbsp;
  <A HREF="http://www.excite.com/search.gw?s=%2Bisi+%2Bdivisions"><B>Excite</B></A>

</BLOCKQUOTE>
<CENTER>

<P><CENTER><!-- Ad Start -->
<a href="/relocate/ad=61447_excite1023_search2celt-468x60.gif|pk=1564947;http://www.excite.com/info/company.html" onMouseOver="window.status='Click Here !';return true" language="Javascript"><img src="http://image.eimg.com/ads/61447_excite1023_search2celt-468x60.gif" ALT="Click Here !" HEIGHT=60 WIDTH=468 BORDER=1><CENTER><FONT SIZE=-1 FACE=geneva,arial COLOR=0000FF> <B> Click Here ! </B></FONT></CENTER></a>
<!-- Ad Stop -->
</CENTER><P>

<FORM ACTION="/search.gw" METHOD=get>
<TABLE BORDER=0 CELLPADDING=0 CELLSPACING=2>
<TR>


<TD ALIGN=middle VALIGN=middle WIDTH=120 ROWSPAN=3>
<TABLE BORDER=1 CELLPADDING=2><TR><TD ALIGN=middle VALIGN=middle HEIGHT=60
WIDTH=90 BGCOLOR="FFFFCC" NOWRAP><A HREF="/magellan/bookmark.html"><FONT SIZE="-1">Bookmark<BR>
Magellan<BR></FONT>NOW!<BR>
<FONT SIZE="-2">click here for details</FONT></A></TD></TR></TABLE></TD>


<TD ALIGN=left VALIGN=top COLSPAN=3 NOWRAP>
<INPUT NAME=search VALUE="+ISI +divisions" SIZE=30>
<INPUT TYPE=IMAGE SRC="/img/magellan/searchbut.gif" BORDER=0 ALIGN="top" VALUE="Search Magellan" WIDTH=67 HEIGHT=30>
</TD></TR>
<TR>
 <TD ALIGN=left>
 <INPUT TYPE=radio NAME=c VALUE=web.guide
    > <FONT SIZE="-1">Reviewed Sites Only</FONT></TD>
 <TD WIDTH=3 ROWSPAN=2></TD>
 <TD ALIGN=left>
  <INPUT TYPE=radio NAME=c VALUE=web.green
        ><FONT SIZE="-1" COLOR=#008C05>Green Light</FONT><FONT SIZE="-1"> sites only</FONT>
 </TD>
</TR>
<TR>
 <TD ALIGN=left><INPUT TYPE=radio NAME=c VALUE=web
   CHECKED  > <FONT SIZE="-1">The Entire Web</FONT>
 <INPUT TYPE=hidden NAME=look VALUE=magellan>
 </TD>

 <TD ALIGN=left><FONT SIZE=-1>
 <A HREF="/magellan/Info/advancedtips.html">Search Tips and Help</A></FONT></TD></TR>
 </TABLE>
<input type=hidden name=sorig value=rpage>
</FORM>


<P><BR>
<CENTER>
<FONT FACE="arial,helvetica" SIZE="-1"><A HREF="http://www.mckinley.com/">Home</A> | <A
HREF="/magellan/Reviews/index.magellan.html">Reviews</A> | <A HREF="/magellan/Info/advancedtips.html">Help</A> | <A HREF="/magellan/Info/addsite.html">Add Site</A><P></FONT>
</FORM>
<P>
<HR>
<FONT SIZE=-1>
Explore <A HREF=/magellan/advertising.html>advertising</A> opportunities with Magellan.<BR>
Copyright &copy;1997 The McKinley Group, Inc., a subsidiary of <A HREF="http://www.excite.com/?Mh-a-t">Excite, Inc.</A>.  All rights reserved.
</FONT>
</CENTER>
<IMG SRC="http://ad.preferences.com/ping.ng;spacedesc=TCExciteData_Excite_1x1_RunOfSite_Any&event=KeywordResults" HEIGHT=1 WIDTH=1 BORDER=0>
</BODY>
</HTML>







