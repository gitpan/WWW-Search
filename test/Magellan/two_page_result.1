<HTML>
<HEAD><TITLE>Search Results</TITLE></HEAD>
<BODY BGCOLOR="#ffffff">

<CENTER>
<TABLE BORDER=0 CELLPADDING=0 CELLSPACING=2>
 <TR>
<TD ALIGN=LEFT VALIGN=top ROWSPAN=4>
<A HREF="http://www.mckinley.com/">
<IMG SRC="/img/magellan/mglobe.gif" BORDER=0 WIDTH=49 HEIGHT=41 ALIGN=left></A>
</TD>
<TD ALIGN=middle VALIGN=top ROWSPAN=4>
<A HREF="http://www.mckinley.com/"><IMG SRC="/img/magellan/magellan.gif" BORDER=0 WIDTH=224 HEIGHT=41 ALIGN=middle ALT="Magellan Internet Guide"></A><BR>
<IMG SRC="/img/magellan/white.gif" HEIGHT=4 WIDTH=1><BR>
<FONT FACE="arial,helvetica" SIZE="-2">
<A HREF="http://www.mckinley.com/">HOME</A> | <A HREF="/magellan/Reviews/index.magellan.html">REVIEWS</A></FONT>
</TD><TD WIDTH=4 ROWSPAN=4></TD>
<TD HEIGHT="2" COLSPAN=3 NOWRAP></TD></TR>

<TR>


<FORM ACTION="/search.gw" METHOD=get>
<TD ALIGN=left VALIGN=top COLSPAN=3 NOWRAP>
<INPUT NAME=search VALUE="+ISI +divisions" SIZE=30>
<INPUT TYPE=IMAGE SRC="/img/magellan/searchbut.gif" BORDER=0 ALIGN="top" VALUE="Search Magellan" WIDTH=67 HEIGHT=30>
</TD></TR>
<TR>
 <TD ALIGN=left NOWRAP>
 <INPUT TYPE=radio NAME=c VALUE=web.guide
    > <FONT SIZE="-1">Reviewed Sites Only</FONT></TD>
 <TD WIDTH=3 ROWSPAN=2></TD>
 <TD ALIGN=left>
  <INPUT TYPE=radio NAME=c VALUE=web.green
        ><FONT SIZE="-1" COLOR=#008C05>Green Light</FONT><FONT SIZE="-1"> sites only</FONT>
 </TD>
</TR>
<TR>
 <TD ALIGN=left><INPUT TYPE=radio NAME=c VALUE=web
   CHECKED  > <FONT SIZE="-1">The Entire Web</FONT>
<INPUT TYPE=hidden NAME=look VALUE=magellan>
</TD>

<!-- End of search and collection section -->


<TD ALIGN=left><FONT SIZE="-1"><A HREF="/magellan/Info/advancedtips.html">Search Tips and Help</A></FONT></TD></TR></TABLE>

<input type=hidden name=sorig value=rpage>
</FORM>
</CENTER>
<P>


 <CENTER><!-- Ad Start -->
<!-- Ad param svcpage=/search.gw keyword=isi+divisions svccobrand=magellan svcname=xsr ip=128.9.160.184 user_agent=WWW::Search/1.023 -->
<a href="/relocate/ad=http://ad.preferences.com/image.ng;spacedesc=ExciteNetwork_Excite_468x60_RunOfSite_Any&time=1998.12.10.18.28.35&ex_cookie=&ex_reg=&TG_NETLOC=xsr/search.gw&TG_COBRAND=magellan|pk=1732784;http://ad.preferences.com/click.ng;spacedesc=ExciteNetwork_Excite_468x60_RunOfSite_Any&time=1998.12.10.18.28.35&ex_cookie=&ex_reg=&TG_NETLOC=xsr/search.gw&TG_COBRAND=magellan" onMouseOver="window.status='Click Here';return true" language="Javascript"><img src="http://ad.preferences.com/image.ng;spacedesc=ExciteNetwork_Excite_468x60_RunOfSite_Any&time=1998.12.10.18.28.35&ex_cookie=&ex_reg=&TG_NETLOC=xsr/search.gw&TG_COBRAND=magellan" ALT="Click Here" HEIGHT=60 WIDTH=468 BORDER=1><CENTER><FONT SIZE=-1 FACE=geneva,arial COLOR=0000FF> <B> Click Here </B></FONT></CENTER></a>
<!-- Ad Stop -->
</CENTER>

<BLOCKQUOTE>

<B>14</B> results returned, ranked by relevance.

<!-- IF statement for Green Light text -->



<FORM ACTION="/search.gw" METHOD=get>
    <INPUT TYPE=hidden NAME=search VALUE="+ISI +divisions">
    <INPUT TYPE=hidden NAME=look VALUE=magellan>
            <INPUT TYPE=submit NAME=prev VALUE="Previous Results">
             <INPUT TYPE=hidden NAME=start VALUE=10>
    <INPUT TYPE=hidden NAME=collection VALUE=web>
            
<!--search results-->

   <HR>
  <B><A HREF="http://www.rsl.ox.ac.uk/guides/rhl/rhl01.html">RHODES HOUSE LIBRARY</A></B>&nbsp;&nbsp;&nbsp;
  <B>37%</B>&nbsp;&nbsp;&nbsp;
  <A HREF="/search.gw?lk=magellan&c=web&qbe=159744&ctag=3664781D&showSummary=true"><font size="-1">[find similar]</A></FONT><BR>
      South Parks Road Oxford OX1 3RG  RHODES HOUSE LIBRARY  OPENING HOURS & CONTACTS   ADMISSION PROCEDURES   MAJOR SUBJECT AREAS, SPECIAL COLLECTIONS & INFORMATION TECHNOLOGY    
    <BR>
  <I> <A HREF="http://www.rsl.ox.ac.uk/guides/rhl/rhl01.html">http://www.rsl.ox.ac.uk/guides/rhl/rhl01.html</A></I>
  <BR>
   <HR>
  <B><A HREF="http://adsabs.harvard.edu/abs_doc/whatsnew.html">What's New with the ADS Abstract Service</A></B>&nbsp;&nbsp;&nbsp;
  <B>37%</B>&nbsp;&nbsp;&nbsp;
  <A HREF="/search.gw?lk=magellan&c=web&qbe=1101624&ctag=3664781D&showSummary=true"><font size="-1">[find similar]</A></FONT><BR>
      Last updated: 19 Nov 1998  The following is new in ADS this month:  1) SAO server downtime this Friday/Saturday 2) CDS Article Mirror Online 3) ApJ and AJ Plates Moved   
    <BR>
  <I> <A HREF="http://adsabs.harvard.edu/abs_doc/whatsnew.html">http://adsabs.harvard.edu/abs_doc/whatsnew.html</A></I>
  <BR>
   <HR>
  <B><A HREF="http://www.angelfire.com/hi/plutonic/afs-faq.html">AFS Frequently Asked Questions</A></B>&nbsp;&nbsp;&nbsp;
  <B>33%</B>&nbsp;&nbsp;&nbsp;
  <A HREF="/search.gw?lk=magellan&c=web&qbe=269336&ctag=3664781D&showSummary=true"><font size="-1">[find similar]</A></FONT><BR>
      AFS FAQ Andrew File System Frequently Asked Questions  
    <BR>
  <I> <A HREF="http://www.angelfire.com/hi/plutonic/afs-faq.html">http://www.angelfire.com/hi/plutonic/afs-faq.html</A></I>
  <BR>
   <HR>
  <B><A HREF="http://www.sil.org/ethnologue/countries/Nigr.html">Ethnologue: Nigeria</A></B>&nbsp;&nbsp;&nbsp;
  <B>31%</B>&nbsp;&nbsp;&nbsp;
  <A HREF="/search.gw?lk=magellan&c=web&qbe=1531701&ctag=3664781D&showSummary=true"><font size="-1">[find similar]</A></FONT><BR>
      Languages of Nigeria.  Part of _Ethnologue: Languages of the World_, 13th Edition; Barbara F.  Grimes, Editor; Summer Institute of Linguistics, 1996.  
    <BR>
  <I> <A HREF="http://www.sil.org/ethnologue/countries/Nigr.html">http://www.sil.org/ethnologue/countries/Nigr.html</A></I>
  <BR>
 <HR>
</FORM>
<FORM ACTION="/search.gw" METHOD=get>
            <INPUT TYPE=submit NAME=prev VALUE="Previous Results">
          <INPUT TYPE=hidden NAME=search VALUE="+ISI +divisions">
    <INPUT TYPE=hidden NAME=look VALUE=magellan>
       <INPUT TYPE=hidden NAME=start VALUE=10>
    <INPUT TYPE=hidden NAME=collection VALUE=web>
          </FORM>

Try your search on:&nbsp;
  <A HREF="http://www.excite.com/search.gw?s=%2Bisi+%2Bdivisions"><B>Excite</B></A>

</BLOCKQUOTE>
<CENTER>

<P><CENTER><!-- Ad Start -->
<a href="/relocate/ad=http://ad.preferences.com/image.ng;spacedesc=ExciteNetwork_Excite_468x60_RunOfSite_Any&time=1998.12.10.18.28.35&ex_cookie=&ex_reg=&TG_NETLOC=xsr/search.gw&TG_COBRAND=magellan|pk=1732784;http://ad.preferences.com/click.ng;spacedesc=ExciteNetwork_Excite_468x60_RunOfSite_Any&time=1998.12.10.18.28.35&ex_cookie=&ex_reg=&TG_NETLOC=xsr/search.gw&TG_COBRAND=magellan" onMouseOver="window.status='Click Here';return true" language="Javascript"><img src="http://ad.preferences.com/image.ng;spacedesc=ExciteNetwork_Excite_468x60_RunOfSite_Any&time=1998.12.10.18.28.35&ex_cookie=&ex_reg=&TG_NETLOC=xsr/search.gw&TG_COBRAND=magellan" ALT="Click Here" HEIGHT=60 WIDTH=468 BORDER=1><CENTER><FONT SIZE=-1 FACE=geneva,arial COLOR=0000FF> <B> Click Here </B></FONT></CENTER></a>
<!-- Ad Stop -->
</CENTER><P>

<FORM ACTION="/search.gw" METHOD=get>
<TABLE BORDER=0 CELLPADDING=0 CELLSPACING=2>
<TR>


<TD ALIGN=middle VALIGN=middle WIDTH=120 ROWSPAN=3>
<TABLE BORDER=1 CELLPADDING=2><TR><TD ALIGN=middle VALIGN=middle HEIGHT=60
WIDTH=90 BGCOLOR="FFFFCC" NOWRAP><A HREF="/magellan/bookmark.html"><FONT SIZE="-1">Bookmark<BR>
Magellan<BR></FONT>NOW!<BR>
<FONT SIZE="-2">click here for details</FONT></A></TD></TR></TABLE></TD>


<TD ALIGN=left VALIGN=top COLSPAN=3 NOWRAP>
<INPUT NAME=search VALUE="+ISI +divisions" SIZE=30>
<INPUT TYPE=IMAGE SRC="/img/magellan/searchbut.gif" BORDER=0 ALIGN="top" VALUE="Search Magellan" WIDTH=67 HEIGHT=30>
</TD></TR>
<TR>
 <TD ALIGN=left>
 <INPUT TYPE=radio NAME=c VALUE=web.guide
    > <FONT SIZE="-1">Reviewed Sites Only</FONT></TD>
 <TD WIDTH=3 ROWSPAN=2></TD>
 <TD ALIGN=left>
  <INPUT TYPE=radio NAME=c VALUE=web.green
        ><FONT SIZE="-1" COLOR=#008C05>Green Light</FONT><FONT SIZE="-1"> sites only</FONT>
 </TD>
</TR>
<TR>
 <TD ALIGN=left><INPUT TYPE=radio NAME=c VALUE=web
   CHECKED  > <FONT SIZE="-1">The Entire Web</FONT>
 <INPUT TYPE=hidden NAME=look VALUE=magellan>
 </TD>

 <TD ALIGN=left><FONT SIZE=-1>
 <A HREF="/magellan/Info/advancedtips.html">Search Tips and Help</A></FONT></TD></TR>
 </TABLE>
<input type=hidden name=sorig value=rpage>
</FORM>


<P><BR>
<CENTER>
<FONT FACE="arial,helvetica" SIZE="-1"><A HREF="http://www.mckinley.com/">Home</A> | <A
HREF="/magellan/Reviews/index.magellan.html">Reviews</A> | <A HREF="/magellan/Info/advancedtips.html">Help</A> | <A HREF="/magellan/Info/addsite.html">Add Site</A><P></FONT>
</FORM>
<P>
<HR>
<FONT SIZE=-1>
Explore <A HREF=/magellan/advertising.html>advertising</A> opportunities with Magellan.<BR>
Copyright &copy;1997 The McKinley Group, Inc., a subsidiary of <A HREF="http://www.excite.com/?Mh-a-t">Excite, Inc.</A>.  All rights reserved.
</FONT>
</CENTER>
<IMG SRC="http://ad.preferences.com/ping.ng;spacedesc=TCExciteData_Excite_1x1_RunOfSite_Any&event=KeywordResults" HEIGHT=1 WIDTH=1 BORDER=0>
</BODY>
</HTML>







