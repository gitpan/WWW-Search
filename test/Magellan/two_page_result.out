http://www.isi.edu/
http://www.isitrans.com/
http://www.nesconsult.com/
http://www.westernreserve.com/
http://www.detnews.com/1996/menu/poli0904.htm
http://hsb.baylor.edu/credits.asp
http://members.tripod.com/~bud_lite/buds_resume.html
http://grognard.com/patch.html
http://www.gospelcom.net/shofar/untilv4i6.html
http://chaos.fullerton.edu/mhslinks.html
http://www.rsl.ox.ac.uk/guides/rhl/rhl01.html
http://adsabs.harvard.edu/abs_doc/whatsnew.html
http://www.angelfire.com/hi/plutonic/afs-faq.html
http://www.sil.org/ethnologue/countries/Nigr.html
